### DISCUSSION TITLE
grass vnc

### DISCUSSION THREAD

**Discussion by warhawk200** [engageub/InternetIncome#181] [Q&A] [answered]
i created 3 grass instances with different proxies , can these be accessed by vnc ? and on which ports ?

**Comment by engageub** [comment]
Hello,
The ports for grass container are not opened.
If they are connected, you will find in your grass dashboard.

Thank you

**Reply by warhawk200** [reply]
i am trying from last two days but only one node shows in dashboard from all three , that is also randomly

so i thought i should check their gui through vnc , but since the ports are closed , i cant be helped 😢

**Reply by engageub** [reply, marked as answer]
Hello,
Grass docker image has been updated 3 hours back.
You can delete and start the script to get the latest docker image.
Grass application usually auto reconnects if there is any issue with connection. 
Also, the connection is VNC and cannot be accessed using browser. 
You will need to use VNC application to access the container. 
It is unlikely that these ports can be accessed outside the VPS with IP address when using proxy.
Serveo.net URL was not being accepted by VNC applications that I have used. 
The author needs to make some configuration changes in order to make the container accessible via browser.


If you would like to see how the GUI looks, you can follow the instructions below.

https://github.com/Trangoul1988/docker-grass-desktop/tree/master

Make sure you have enough resources, else your application may not login automatically. 

Thank you

**Reply by warhawk200** [reply]
Thanks 🙌

**Reply by warhawk200** [reply]
> Hello, Grass docker image has been updated 3 hours back. You can delete and start the script to get the latest docker image. Grass application usually auto reconnects if there is any issue with connection. Also, the connection is VNC and cannot be accessed using browser. You will need to use VNC application to access the container. It is unlikely that these ports can be accessed outside the VPS with IP address when using proxy. Serveo.net URL was not being accepted by VNC applications that I have used. The author needs to make some configuration changes in order to make the container accessible via browser.
> 
> If you would like to see how the GUI looks, you can follow the instructions below.
> 
> https://github.com/Trangoul1988/docker-grass-desktop/tree/master
> 
> Make sure you have enough resources, else your application may not login automatically.
> 
> Thank you

Update: Looks like at grass dashboard many ip show offline but they are giving points and uptime is also being counted 😊

**Reply by engageub** [reply]
Yes, there may be delays in grass dashboard updates. Sometimes, it takes up to 48 hours to update the data.
